webclient/app/src/app/commons/entityDialog/EntityDialog.jsx

```diff
@@ -55,11 +54,34 @@ function EntityDialog(props) {
                 hasError = true;
             }
 
-        })
+        });
+
+        entity.relationships?.forEach(relationship => {
+            if (!RegexConfig.relationship.test(relationship.relationshipName)
+                || !RegexConfig.relationship.test(relationship.otherEntityRelationshipName)
+                || !RegexConfig.entityTitle.test(relationship.otherEntityName)) {
+                hasError = true;
+            }
+        });
         setHasFormError(hasError);
 
+
     }, [entity])
 
+
+    function getTargetEntities() {
+        let newTargetEntities = [];
+        if (entities && entities.length > 1) {
```

Comment:
```suggestion
        if (entities?.length > 1) {
```
This should work as well